src/rocksdb.rs

```diff
@@ -52,10 +52,13 @@ impl Drop for CFHandle {
     }
 }
 
-fn ensure_default_cf_exists<'a>(list: &mut Vec<ColumnFamilyDescriptor<'a>>) {
+fn ensure_default_cf_exists<'a>(list: &mut Vec<ColumnFamilyDescriptor<'a>>, ttls: &mut Vec<i32>) {
     let contains = list.iter().any(|ref cf| cf.is_default());
     if !contains {
         list.push(ColumnFamilyDescriptor::default());
+        if ttls.len() < list.len() {
```

Comment:
Only the length of the original `list` equals to the length of the original `ttls`, we should add `ttl` for default column family

Reply:
what if user just don't use ttl open here?

Reply:
yes, this will add ttl, but it doesn't affect for `with_ttl` would be false, and not use ttl.  This logic is ok here, but would not be clear, so I will add condition test here.

Reply:
So that we can simplify `with_ttl` too, it's a little bit confusing. Do you think it's better like:
```
let with_ttl = if ttls_vec.len() > 0 {
    if ttls_vec.len() == cf_names.len() {
        true
    } else {
        return Err("the length of ttls not equal to length of cfs".to_owned());
    }
} else {
    false
}
```

Reply:
yes, It's better. I will change to like this `let with_ttl = if ttls.len() > 0 {
    if ttls.len() == cfds.len() && ttls_vec.len() == cf_names.len() {
        true
    } else {
        return Err("the length of ttls not equal to length of cfs".to_owned());
    }
} else {
    false
}` to make sure be not confusing.

Reply:
I have commited it again.

Reply:
How about change the condition here FROM:
```
if ttls.len() < list.len() {
```
TO
```
if ttls.len() > 0 {
```
So that we can get rid of judging if `ttls.len() > 0` when deal with `with_ttl`

Reply:
Yes, this way is more clear. I have changed and committed it.